ddtrace/appsec/_asm_request_context.py

```diff
@@ -65,31 +51,36 @@ def __init__(self, active=False):  # type: (bool) -> None
         self.addresses_sent = set()  # type: set[str]
 
 
-_ASM = contextvars.ContextVar("ASM_contextvar", default=ASM_Environment())
+def _get_asm_context():
```

Comment:
Yes, that's a fine default. When that doesn't suit a product's needs, it can create and manage its own `ExecutionContext`, a child of the root context.